modules/aerodyn/src/AeroDyn_Types.f90

```diff
@@ -226,7 +227,7 @@ MODULE AeroDyn_Types
     CHARACTER(1024)  :: FVWFileName      !< FVW input filename [quoted string]
     CHARACTER(1024) , DIMENSION(:), ALLOCATABLE  :: AFNames      !< Airfoil file names (NumAF lines) [quoted strings]
     LOGICAL  :: UseBlCm      !< Include aerodynamic pitching moment in calculations? [flag]
-    LOGICAL  :: SumPrint      !< Generate a summary file listing input options and interpolated properties to "<rootname>.AD.sum"? [flag]
+    LOGICAL  :: SumPrint      !< Generate a summary file listing input options and interpolated properties to  [<rootname>.AD.sum"?"]
```

Comment:
The comments/units here are a little bit off... probably due to the way quotation marks are used in the registry input file.